movies/movies/TableView/AboutMovieViewController.swift

```diff
@@ -42,49 +51,6 @@ final class AboutMovieViewController: UIViewController {
         movieTable.delegate = self
         movieTable.dataSource = self
     }
-
-    private func fetchDataAboutMovie(cell: AboutMovieTableViewCell, indexPath: IndexPath) {
-        guard let url =
-            URL(
-                string: "https://api.themoviedb.org/3/movie/" + "\(idMovie)" +
-                    "?api_key=90f917324ecb224bd306de1b97b17591&language=ru-RU"
-            )
-        else { return }
-        let session = URLSession.shared
-        session.dataTask(with: url) { data, _, _ in
-            guard let data = data
-            else { return }
-            do {
-                let decoder = JSONDecoder()
-                decoder.keyDecodingStrategy = .convertFromSnakeCase
-                self.aboutMovie = try decoder.decode(Result.self, from: data)
-                DispatchQueue.main.async { [weak self] in
-                    guard let self = self else { return }
-                    cell.aboutMoviesLabel.text = self.aboutMovie?.overview
-                    guard let release = self.aboutMovie?.releaseDate else { return }
-                    cell.releaseDateLabel.text = "Даты выхода: \(release))"
-                    guard let mark = self.aboutMovie?.voteAverage else { return }
-                    cell.voteLabel.text = "Оценка \(mark)"
-                    cell.titleMoviesLabel.text = self.aboutMovie?.title
-                    self.movieTable.reloadData()
-                }
-            } catch {
-                print(error)
-            }
-        }.resume()
-        guard let addresImage = aboutMovie?.posterPath else { return }
-        guard let url = URL(string: "https://image.tmdb.org/t/p/w500" + addresImage) else { return }
-        URLSession.shared.dataTask(with: url) { data, _, _ in
-            if let data = data, let image = UIImage(data: data) {
-                DispatchQueue.main.async { [weak self] in
-                    guard let self = self else { return }
-                    cell.posterMovieImageView.image = image
-                    cell.posterMovieImageView.layer.cornerRadius = 10
-                    self.movieTable.reloadData()
-                }
-            }
-        }.resume()
-    }
 }
 
 extension AboutMovieViewController: UITableViewDelegate, UITableViewDataSource {
```

Comment:
```suggestion
extension AboutMovieViewController: UITableViewDelegate {
```
каждый протокол в отдельном экстеншене